src/hotspot/share/c1/c1_Runtime1.cpp

```diff
@@ -177,11 +177,11 @@ static bool caller_is_deopted() {
 // Stress deoptimization
 static void deopt_caller() {
   if ( !caller_is_deopted()) {
-    JavaThread* thread = JavaThread::current();
-    RegisterMap reg_map(thread, false);
-    frame runtime_frame = thread->last_frame();
+    JavaThread* current = JavaThread::current();
```

Comment:
It looks like both of these functions could be passed JavaThread from the callers.  You could leave this as a cleanup though.  It would eliminate two explicit JavaThread::current calls.

Reply:
That's a good additional cleanup - thanks. Fixed.